recipes/armadillo/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "7.600.2" %}
+
+{% set variant = "openblas" %}
+
+package:
+    name: armadillo
+    version: {{ version }}
+
+source:
+    fn: armadillo-{{ version }}.tar.xz
+    url: http://sourceforge.net/projects/arma/files/armadillo-{{ version }}.tar.xz
+    sha256: 6790d5e6b41fcac6733632a9c3775239806d00178886226dec3f986a884f4c2d
+    patches:
+        # have to add additional ${PREFIX}/lib etc... to tests/Makefile
+        - make_tests.patch  # [linux]
+        - make_tests_osx.patch  # [osx]
+        # modify CMakeLists to use OpenBLAS build rather than Accelrate
+        - CMakeLists.patch  # [osx]
+
+build:
+  number: 200
+  skip: True  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake
+    # optional items below here
+    - pkg-config
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - lapack
+    - arpack
+    - superlu 5*
+    # - hdf5 1.8.17|1.8.17.
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.19|0.2.19.*
+    - lapack
+    - arpack
+    - superlu 5*
+    # - hdf5 1.8.17|1.8.17.
+
+test:
+  commands:
+    - test -f "${PREFIX}/include/armadillo"
+    - test -d "${PREFIX}/include/armadillo_bits"
+    - test -f "${PREFIX}/lib/libarmadillo.so"  # [linux]
+    - test -f "${PREFIX}/lib/libarmadillo.dylib"  # [osx]
+
+about:
+    home: http://arma.sourceforge.net
+    license: Mozilla Public 2.0 (MPL 2.0)
```

Comment:
Normally the short form is preferred. So simply MPL 2.0 is fine.

Also please add `license_file: LICENSE.txt` so that we can include the license with the conda package.